netpyne_ui/netpyne_geppetto.py

```diff
@@ -317,6 +317,15 @@ def getAvailablePlots(self):
         plots  = ["plotRaster", "plotSpikeHist", "plotSpikeStats","plotRatePSD", "plotTraces", "plotLFP", "plotShape", "plot2Dnet", "plotConn", "granger"]
         return [plot for plot in plots if plot not in simConfig.analysis.keys()]
 
+    def deleteParam(self, paramToDel):
+        logging.debug("checking if netParams."+paramToDel+" is not null")
```

Comment:
If you want to keep the logs maybe try to explain you are about to delete a param or sth like that and, being a bit picky, use capital letter on init